### ISSUE TITLE
Slide bug with touch scroll

### ISSUE THREAD

**Issue by TheDelta** [MurhafSousli/ngx-gallery#465]
#### What is the expected behavior?

When I touch the image, it should not move.
When I touch and move down / up, it shouldn't try to switch to the next item.

#### What is the current behavior?

Once i touch and move, it will try to slide to the next

#### What are the steps to reproduce?

https://stackblitz.com/edit/ngx-gallery

open the starter with a smartphone and try to scroll down in google chrome.

Another way to see the odd slide glitch is to open the lightbox, an touch and move up/down at the right side (below the arrow button).

#### What is the use-case or motivation for changing an existing behavior?

To support a nice user experience, it slide unless a tiny threshold is reached to left/right slide.

#### Which versions are you using for the following packages?

(the ones in the StackBlitz starter)

#### Is there anything else we should know?

Eventually this makes sense to fix with the replacement of hammerjs (https://github.com/MurhafSousli/ngx-gallery/pull/452) or it might be already fixed with it.

**Comment by davide1995** [comment]
Same problem here!
In my project, I am experiencing this problem with scrolling through a gallery and I have noticed that the https://ngx-gallery.netlify.app/#/gallerize demo has the same problem. As soon as I scroll with my finger on the gallery, the next image bounces back and then shows the current one again. Below is a video of the problem encountered

https://user-images.githubusercontent.com/21131407/193148968-1ae49284-07d4-45da-ad49-d22326182f14.mp4

**Comment by davide1995** [comment]
The same problem occurs when resizing the window with a PC as shown in the below video. There is probably some problem with the gestures

https://user-images.githubusercontent.com/21131407/193155181-b98557bf-af64-4015-8dbd-240876a23533.mp4

**Comment by MurhafSousli** [comment]
@TheDelta  #452 doesn't solve the problem, this is related to the browser nature with the touch actions

Check it here https://developer.mozilla.org/en-US/docs/Web/CSS/touch-action

Workaround is to add the following in `style.scss`

```scss
gallery-slider {
  touch-action: pan-x;  // for horizontal slider
}
```

Keep in mind that this will block scrolling vertically in that region!

**Comment by MurhafSousli** [comment]
This is now fixed in v7.1.1, thanks for opening the issue!

I added a new option  `reserveGesturesAction` which blocks the page from accidently scrolling when user is not accurately sliding the gallery, this option is `true` by default when using the lightbox.

**Comment by MurhafSousli** [comment]
> The same problem occurs when resizing the window with a PC as shown in the below video. There is probably some problem with the gestures
> 
>  Untitled.mp4

Actually this is not related to the same bug, but it is valid issue

**Comment by davide1995** [comment]
> > The same problem occurs when resizing the window with a PC as shown in the below video. There is probably some problem with the gestures
> > Untitled.mp4
> 
> Actually this is not related to the same bug, but it is valid issue

Should I write an other issue just for this problem?

**Comment by MurhafSousli** [comment]
@davide1995 In v7.1.2 it was enhanced, now it goes back to the correct slide, but that effect of sliding won't be fixed easily, at least to my knowledge. check out how it behaves with the new version, I think it is acceptable now.

**Comment by davide1995** [comment]
> @davide1995 In v7.1.2 it was enhanced, now it goes back to the correct slide, but that effect of sliding won't be fixed easily, at least to my knowledge. check out how it behaves with the new version, I think it is acceptable now.

Yes, the scroll issue was solved with version 7.1.2 thank you. In my opinion, however, I think that by resizing the window the user should remain focused on the current side. But no problem, it's just a detail. Otherwise if I have time I'll propose a solution with a pull request
